README.md

```diff
@@ -1120,6 +1122,356 @@ A component used to render JSON data based on a schema and a UI schema.
 | `extraContext` | `object` | - | - | Extra context used by `json-e` when transforming the UI schema. |
 | `validate` | `boolean` | - | - | If set, the `value` will be validated against the `schema` and any error will be displayed at the top of the rendered output. Useful for debugging. |
 
+#### UI Schema fields
+
+The UI schema for any value may consist of any of the following fields:
+
+| Field | Description |
+|-------|-------------|
+| `ui:title` | The title of the data field.<br><br>If not set, the value of `schema.title` is used. If `schema.title` is not set, the value's key is used, in 'Start Case'.<br><br>Set `'ui:title': null` to hide the title. |
+| `ui:description` | The description of the data field.<br><br>If not set, the value of `schema.description` is used.<br><br>Set `'ui:description': null` to hide the description. |
+| `ui:widget` | Specifies the widget to use to render the data |
+| `ui:order` | For values of type `object`, this field can be used to specify the order in which to render the object's properties.<br><br>e.g. `'ui:order': [ 'propB', 'propA' ]` |
+| `ui:value` | Can be used to override/transform the data value. Particularly useful when used with [json-e](https://json-e.js.org/) transforms.<br><br>e.g. `'ui:value': 'Hello ${source}'` |
+| `ui:options` | An object that can be provided to define additional UI options specific to the widget used to render the data value. For more details see the [Widgets README](./widgets/README.md#ui-options). |
+| `items` | If the value is an array, use the `items` field to specify the UI schema of all items in the array. This is where `json-e` is particularly useful when combined with `${source}` to get the value of the item. |
+
+Note that all fields in the UI schema can be expressed using [json-e](https://json-e.js.org/) syntax.
+
+##### Transforming nested UI schemas
+
+Each 'level' of a nested UI schema is transformed independently using json-e. 
+
+Consider the following example:
+
+```javascript
+const value = {
+  links: [ 'link1', 'link2' ]
+}
+
+const uiSchema = {
+  links: {
+    'ui:widget': 'Link',
+    items: {
+      'ui:options': {
+        href: 'https://mysite.com/${source}'
+      }
+    }
+  }
+}
+```
+
+In this example a json-e tranform is run for each item in the `links` array, allowing a unique value of `ui:options.href` to be set for each link in the array.
```

Comment:
NIT: small typo `tranform`
```suggestion
In this example a json-e transform is run for each item in the `links` array, allowing a unique value of `ui:options.href` to be set for each link in the array.
```

Reply:
Woah - eagle-eyes!! Impressive spot (or did you use Grammarly...? 😆). Fixed now.

Reply:
@grahammcculloch :joy: I actually spotted this one "raw"